### PR TITLE
Update build and tweak spacing for lists/headings

### PR DESCRIPTION
**Pull request by matt-in-a-hat**
Increases spacing after ol/ul
Increases spacing above headings
Fixes issues getting build to work by updating modules and running in recent node

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,9 +40,16 @@
...
+    "webpack": "^5.8.0",
+    "webpack-stream": "^6.1.1"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "browserslist": [
+    "defaults",
+    ">0.1%"
+  ]
```
Comment (markstuart) [inline package.json:54]: What caused `browserslist to get added? Was it intentional?

File: gulpfile.js [hunk 2]
```diff
@@ -23,16 +23,13 @@ const async = require('async')
...
+const notifier = require('node-notifier')
 
 const config = {
   path: {
     css: './dist/css',
     js: './dist/js'
-  },
-  browsers: [
-    'last 5 versions'
-  ]
```
Comment (markstuart) [inline gulpfile.js:35]: Ah, browserslist was a replacement for this gulp setting I guess.

**Review by markstuart — APPROVED** [review]
Looks good to me 👍

What would make this sort of styling PR even better would be before/after screenshots 😉

**Comment by markstuart** [comment]
I have just noticed that you said you've added spacing after lists... but it looks to me like you've replaced a `/ 2` with a `* 0.5`, which is equivalent, right?

**Comment by matt-in-a-hat** [comment]
That would be correct, but I replaced it with * 1.5 😆

**Review by jooleearr — APPROVED** [review]
LGTM 👌

**Pull request merged**